src/app/find-help/accommodation-search/accommodation-search.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { AccommodationQuery } from './accommodation-search-form/accommodation-search-form.component';
+import { Pagable } from '../../api/pagable';
+import { OfferLocation } from '../../api/types';
+
+interface AccommodationOffer {
+  id: number;
+  title: string;
+  description: string;
+  location: OfferLocation;
+  guests: number;
+  lengthOfStay: string;
+  hostLanguage: string[];
+}
+@Component({
+  selector: 'app-accommodation-search',
+  templateUrl: './accommodation-search.component.html',
+  styleUrls: ['./accommodation-search.component.scss'],
+})
+export class AccommodationSearchComponent {
+  results: AccommodationOffer[] = [];
+  total: number = NaN;
+  loading = false;
+  constructor(private http: HttpClient) {}
+
+  search(query: AccommodationQuery) {
+    this.loading = true;
+    this.http
```

Comment:
We should generate services from the swagger file.

Reply:
I suggest setting that up in another PR